cpp/test/summit_gpu.sh

```diff
@@ -10,6 +10,9 @@ export CC=gcc
 export CXX=g++
 export CXXFLAGS="-O3"
 export ARCH="CUDA"
-export CUDA_ARCH="-arch sm_70 --std=c++14 --use_fast_math -O3"
+export YAKL_CUDA_FLAGS="-arch sm_70 --std=c++14 --use_fast_math -O3"
```

Comment:
Can you remove the `--stc=c++14` flag? That's handled internally now.

Reply:
Done.